Lib/test/test_typing.py

```diff
@@ -592,10 +592,9 @@ def test_no_duplicates_if_replacement_not_in_templates(self):
 class GenericAliasSubstitutionTests(BaseTestCase):
     """Tests for type variable substitution in generic aliases.
 
-    Note that the expected results here are tentative, based on a
-    still-being-worked-out spec for what we allow at runtime (given that
-    implementation of *full* substitution logic at runtime would add too much
-    complexity to typing.py). This spec is currently being discussed at
+    For variadic cases, these tests should be regarded as the source of truth,
+    since we hadn't realised the full complexity of variadic substitution
+    at the time of finalizing PEP 646. For full discussion, see
```

Comment:
If the SC doesn't object maybe we could update the PEP with a more exact specification? Since this was discovered during the beta test period it makes sense to me to update the PEP. Storing a part of a specification permanently in an issue seems iffy to me (plenty of tests refer to the issue that caused the test to be added, but the wording here makes me think that the situation here is different -- this is effectively part of the specification).

Reply:
Sure - if the SC is fine with it, I'd be happy to write this up in the PEP properly.

Reply:
@mrahtz From the SC issue I gather the following:
- We should add these tests (they'll land in 3.11.1, that's fine)
- We (you) should write the PEP update and submit it to the SC for formal approval

I expect the SC approval for the PEP updates should be easy to obtain.

Reply:
Ok, great. I'll work on the PEP updates as soon as I have some time.